core/src/test/java/org/javarosa/core/model/test/FormDefTest.java

```diff
@@ -432,7 +440,49 @@ public void testSimilarBindConditionsAreDistinguished() throws Exception {
         }
     }
 
-
+    // regression test for when we weren't decrementing multiplicities correctly when repeats were deleted
+    @Test
+    public void testDeleteRepeatMultiplicities() throws IOException {
+        FormParseInit fpi = new FormParseInit("/multiple_repeats.xml");
+        FormEntryController fec = initFormEntry(fpi, "en");
+        fec.stepToNextEvent();
+        fec.newRepeat();
+        fec.stepToNextEvent();
+        fec.answerQuestion(new StringData("Repeat 1/1"));
+        fec.stepToNextEvent();
+        fec.answerQuestion(new StringData("Repeat 1/2"));
+        fec.stepToNextEvent();
+        fec.newRepeat();
+        fec.stepToNextEvent();
+        fec.answerQuestion(new StringData("Repeat 2/1"));
```

Comment:
Instead of `1/1` and `2/1` could you say something like `1st repeat, 1st entry` and `1st repeat, 2nd entry`